packages/ms-vc-api-issuer/__tests__/shared/vcApiIssuerClientCredentialAgentLogic.ts

```diff
@@ -0,0 +1,36 @@
+import { TAgent, createAgent } from '@veramo/core'
+import { IMsVcApiIssuer, MsAuthenticationTypeEnum, MsVcApiIssuer } from '../../src'
+
+type ConfiguredAgent = TAgent<IMsVcApiIssuer>
+
+export default (testContext: { getAgent: () => ConfiguredAgent; setup: () => Promise<boolean>; tearDown: () => Promise<boolean> }) => {
+  describe('Issuer Agent Plugin', () => {
+    let agent: TAgent<IMsVcApiIssuer>
+
+    beforeAll(async () => {
+      await testContext.setup()
+      agent = createAgent({
+        plugins:[
+          new MsVcApiIssuer({
+            authenticationType: MsAuthenticationTypeEnum.ClientCredential,
+            authenticationArgs: {
+              azClientId: '<client_id>',
+              azClientSecret:'<client_secret>',
+              azTenantId: '<tenant_id>',
+              credentialManifest:'<credential_manifest>'
```

Comment:
Hmzz, not sure whether this is going to work in a multitenant setup, as you now need multiple agent instances per manifest and per Azure tenant. I think we might want this as a default for a single tenant scenario, but we need to be able to override the tenant per method invocation (so move it to an options object). The manifest needs to move in all cases. Since a lot of parties will want to issue multiple different credentials